components/src/lib/Dialog.css.ts

```
@@ -9,33 +9,59 @@ export const StyledOverlay = styled.div`
+        return `
+          width: 320px;
+          max-width: 90%;
+          padding: 16px;
+        `;
+      case 'large':
+        return `
+          width: 800px;
+          max-width: 90%;
+          padding: 24px;
+        `;
+      case 'medium':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)